### PR TITLE
New: Validations on the whole invoicing process

### PR DESCRIPTION
**Pull request by mroig**
- [x] While importing F1
  - [x] On phase 1
  - [x] On phase 2
  - [x] On phase 3
  - [x] On phase 4
- [x] Validations on the "lot"
- [x] Validations on invoice creation

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

File: docs/comer/facturacion.md [hunk 1]
```diff
@@ -127,33 +127,161 @@ Procediment d'importació
 En processar un fitxer xml, ja sigui provinent d'un zip o individualment, el
 mòdul realitza les següents accions:
 
-1. Facturar a partir de les lectures del fitxer xml.
-2. Generar una factura a partir de les línies de factura del fitxer xml.
-3. Comparar ambdues factures.
+1- Pujar el fitxer F1 al servidor
```
Comment (jaumef) [inline docs/comer/facturacion.md]: Compte!
Al posar la numeració amb guions, t'ho ha posat tot amb una sola línia

http://builds.gisce.net/powerp-docs/powerp_add_validations/ca/comer/facturacion/#procediment-dimportacio

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

File: docs/comer/facturacion.md (hunk 1 above)
Comment (mroig) [inline docs/comer/facturacion.md]: Solucionat :+1:

**Comment by jaumef** [comment]
No es va mergejar correctament amb master, els camps del messages.po:
"POT-Creation-Date: \n"
"PO-Revision-Date: \n"
Han de tenir una data correcta:
- teoricament major a l'anterior edició en el cas del PO
- Ha de ser major la del PO que la del POT

**Pots arreclar-ho utilitzant `./translate.sh`**

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_validations/

**Pull request merged**